mvcc/backend/backend.go

```diff
@@ -165,31 +166,40 @@ func (b *backend) ForceCommit() {
 }
 
 func (b *backend) Snapshot() Snapshot {
+	b.batchTx.Commit()
+
+	b.mu.RLock()
+	defer b.mu.RUnlock()
+	tx, err := b.db.Begin(false)
+	if err != nil {
+		plog.Fatalf("cannot begin tx (%s)", err)
+	}
+
 	stopc, donec := make(chan struct{}), make(chan struct{})
+	dbBytes := tx.Size()
 	go func() {
 		defer close(donec)
+		// snapshotWarningTimeout is based on transferring snapshot data over a 1 gigabit/s connection
+		// assuming a min tcp throughput of 100MB/s.
+		var throughputBytes int64 = 100 * 1024 * 1014
```

Comment:
Huh? The concern is int/int will truncate to 0 for cases where float/float won't truncate to 0. Not concerned about sendRateBytes being 0 and therefore causing divide by zero.